src/systems/include/mp-units/systems/si/unit_symbols.h

```diff
@@ -474,6 +474,32 @@ inline constexpr auto YF = yotta<farad>;
 inline constexpr auto RF = ronna<farad>;
 inline constexpr auto QF = quetta<farad>;
 
+inline constexpr auto qohm = quecto<si::ohm>;
+inline constexpr auto rohm = ronto<si::ohm>;
+inline constexpr auto yohm = yocto<si::ohm>;
+inline constexpr auto zohm = zepto<si::ohm>;
+inline constexpr auto aohm = atto<si::ohm>;
+inline constexpr auto fohm = femto<si::ohm>;
+inline constexpr auto pohm = pico<si::ohm>;
+inline constexpr auto nohm = nano<si::ohm>;
+inline constexpr auto uohm = micro<si::ohm>;
+inline constexpr auto mohm = milli<si::ohm>;
+inline constexpr auto cohm = centi<si::ohm>;
+inline constexpr auto dohm = deci<si::ohm>;
+inline constexpr auto ohm = si::ohm;
```

Comment:
I did not add those exactly because of that. If someone does a using-directive on both `si` and `si::unit_symbols,` then `ohm` will be ambiguous. I still am not sure it that is an issue or not. @JohelEGP, do you have an opinion here?

Reply:
Hm. Actually, I'm doing that myself: 
```c++
namespace some_relevant_namespace_for_my_code {

namespace u {
using namespace mp_units;
using namespace mp_units::si;
using namespace mp_units::isq;
using namespace mp_units::si::unit_symbols;
}

}
```

Maybe I never use plain `ohm` anywhere? I would definitely want to have some kind of prefixed ohm units somewhere - maybe we keep the rest and skip the plain `ohm`? Altentatively, the v1 "references" `kR` etc... did not have this isssue, but are obvisously somewhat non-standard.